src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSpec.scala

```diff
@@ -439,7 +439,7 @@ class LibraryServiceSpec extends BaseServiceSpec with FreeSpecLike with LibraryS
         "NCTRL" -> JsBoolean(false),
         "GRU" -> JsBoolean(false),
         "HMB" -> JsBoolean(false),
-        "RS-POP" -> JsArray(),
+        //"RS-POP" -> JsArray(),
```

Comment:
Okay... I finally asked and got the story about RS-POP. Probably worth removing the code instead of commenting out?